src/protego.py

```diff
@@ -307,6 +307,8 @@ def __init__(self):
     @classmethod
     def parse(cls, content):
         o = cls()
+        if isinstance(content, bytes):
+            content = content.decode()
```

Comment:
Why is `utf-8` is considered here?
Converting from bytes should be done on HTTP level as encoding is known at that stage.

Reply:
Maybe I was too rush when requesting https://github.com/scrapy/protego/issues/16?

Instead of adding support, should we raise a `ValueError` if the input is not `str`?

Reply:
Good point @whalebot-helmsman! I changed the PR accordingly